webpack.js

```diff
@@ -1,33 +1,22 @@
 const webpackConfig = require('@nextcloud/webpack-vue-config')
 const path = require('path')
-const { merge } = require('webpack-merge')
 
-const config = {
-	entry: {
-		collections: path.join(__dirname, 'src', 'init-collections.js'),
-		dashboard: path.join(__dirname, 'src', 'init-dashboard.js'),
-		calendar: path.join(__dirname, 'src', 'init-calendar.js'),
-		talk: path.join(__dirname, 'src', 'init-talk.js'),
-	},
-	output: {
-		filename: '[name].js',
-		jsonpFunction: 'webpackJsonpOCADeck',
-		chunkFilename: '[name].js?v=[contenthash]',
-	},
-	resolve: {
-		extensions: ['*', '.js', '.vue', '.json'],
-		modules: [
-			path.resolve(__dirname, 'node_modules'),
-			'node_modules',
-		],
-	},
-	stats: {
-		context: path.resolve(__dirname, 'src'),
-		assets: true,
-		entrypoints: true,
-		chunks: true,
-		modules: true
-	}
+webpackConfig.entry = {
+	...webpackConfig.entry,
+	collections: path.join(__dirname, 'src', 'init-collections.js'),
+	dashboard: path.join(__dirname, 'src', 'init-dashboard.js'),
+	calendar: path.join(__dirname, 'src', 'init-calendar.js'),
+	talk: path.join(__dirname, 'src', 'init-talk.js'),
 }
 
-module.exports = merge(webpackConfig, config)
+webpackConfig.resolve.extensions.push('.json')
```

Comment:
Do you actually import json files?

Reply:
Not that i remember, but let me drop it since removing doesn't have any effect on the resulting bundle :grin: